.github/workflows/publish.yml

```diff
@@ -0,0 +1,47 @@
+name: Upload Package to Pypi and TestPyPI
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
This should be "master" I think, as that's the branch name here.

Reply:
Good catch, forgot this is still using the old naming, will update today